adapters/outgoing/scrapers/src/main/java/be/xplore/pricescraper/item/AhBeScraper.java

```diff
@@ -1,4 +1,4 @@
-package be.xplore.pricescraper.utils.scrapers;
```

Comment:
missing the ball a bit by adding all scrapers in the same module, then it becomes an all or nothing situation anymore and defeats a bit the purpose

imagine all the scrapers as usb devices you can plug any of them in just the AhBeScraper or the carrefourscraper or all of them


what you did now it just have just one usb device which does everything at once

if this analogy makes sens 0:)